README.md

```diff
@@ -0,0 +1,103 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# The Budget
+>This is a mobile web application build with Ruby on Rails. Users can keep track of their expenses, and add them to their desired categories. 
+
+## Built With
+
+- Ruby
+- Ruby on Rails
+- Git
+- Postgresql
+
```

Comment:
- [ ] Kindly ensure that the link to the deployed project is present in the README